### PR TITLE
add translateMask method (also realphabetize to keep things tidy)

### PR DESCRIPTION
**Pull request by jebeck**
Here's the new method @jh-bate 

LMK if you think the tests and error handling are as desired and/or adequate.

### CODE REVIEW

File: lib/datetimeWrapper.js [hunk 1]
```diff
@@ -311,32 +331,32 @@ var datetimeWrapper = function(moment) {
...
+     * @param {String} Input date and/or time string
+     * @param {String} Input's format mask
+     * @param {String} Desired output format mask
      *
-     * @return {string} an ISO-8601-formatted zulu timestamp N days from now
+     * @return {String} Reformatted date and/or time string
      */
-    futureDate: function(ndays) {
-      return moment().utc().add(ndays, 'days').toISOString();
+    translateMask: function(timestr, MASK1, MASK2) {
```
Comment (jh-bate) [inline lib/datetimeWrapper.js]: rename params fromMask, toMask  or inputMask, outputMask ??

**Comment by jh-bate** [comment]
tests look all good, just wondered about param naming for usability

File: lib/datetimeWrapper.js (hunk 1 above)
Comment (jebeck) [inline lib/datetimeWrapper.js]: yeah, will do, good idea

**Comment by jh-bate** [comment]
LGTM will merge once built

**Pull request merged**